src/clusterfuzz/_internal/common/testcase_utils.py

```diff
@@ -61,15 +60,30 @@ def emit_testcase_triage_duration_metric(testcase_id: int, step: str):
                  ' failed to emit TESTCASE_UPLOAD_TRIAGE_DURATION metric.')
     return
 
+  from_fuzzer = not get_testcase_upload_metadata(testcase_id)
+
+  assert step in [
```

Comment:
So we can throw an assertion failure if our metrics collection isn't working? I don't think this is a good idea.

Reply:
+1 to removing this.